### ISSUE TITLE
Python deps keeps the == next to the version number in title

### ISSUE THREAD

**Issue by OmgImAlexis** [renovatebot/renovate#2228]
**What Renovate type are you using?**
Renovate GitHub App

**Additional context**
Please see https://github.com/pymedusa/Medusa/pull/4591

**Comment by rarkins** [comment]
@OmgImAlexis this is deliberate.. for now. Do you think Python users would prefer it to be like `v1.12.0` ?

**Comment by OmgImAlexis** [comment]
PING @sharkykh @medariox

**Comment by sharkykh** [comment]
@rarkins 
I think it depends on the `rangeStrategy` setting.
This is my first time looking at the config, but if Python projects define `rangeStrategy` to `pin`,
then it should be just like Node.js dependencies are handled.

Python's version markers are simpler than the ones used in Node.js.
`==`, `>=`, `<=`, `<`, `>`, `!=`, and they are delimited by commas, for example:
```
vcrpy>=1.11.0,<2
```

This is the simplest form, obviously it can have more marker types, each type is delimited by a semi-colon. https://www.python.org/dev/peps/pep-0508/#examples

**Comment by rarkins** [comment]
Even if users have `rangeStrategy` set to *not* `pin`, we will still get updates like this if their existing versions are pinned.

So the question essentially is: if the existing version of "foo" is `==1.0.0` and `1.0.1` is available, should the commit message and PR title say "Update dependency foo to ==1.0.1" or "Update dependency foo to v1.0.1"? Obviously the `requirements.txt` itself should have the `==` in regardless.

**Comment by sharkykh** [comment]
> if the existing version of "foo" is ==1.0.0 and 1.0.1 is available
For this specific case, I'd say "Update dependency foo to v1.0.1" is okay.

**Comment by medariox** [comment]
I agree with @sharky, there is no real advantage to keep the `==` and it looks weird/unnecessary (unless I'm missing something).
